### PR TITLE
Foreman 3.5.0 translations

### PR DESCRIPTION
**Pull request by ekohl**
This includes some fixes to source strings that were malformed. Sadly they mostly landed in 3.1 - 3.3 and nothing in the current dev cycle. We should have a CI check for this to avoid it in the future.

### CODE REVIEW

**Comment by ekohl** [comment]
@MariaAga and @jeremylenz git blame points to you (and Amir, but he left the project). Would you mind checking if my changes are correct?

**Review by jeremylenz — DISMISSED** [review]
I wasn't aware that these practices were frowned upon. I'd think there'd be lint rules that could yell at us about this..

anyway, LGTM :+1:

**Comment by jeremylenz** [comment]
speaking of lint rules, the current failures may shed some light :)

**Comment by ekohl** [comment]
> I wasn't aware that these practices were frowned upon. I'd think there'd be lint rules that could yell at us about this..

My thoughts exactly. I was surprised we don't have them because when you run the update task it yells about malformed strings. https://projects.theforeman.org/projects/foreman/wiki/Translating#Translating-for-developers has also been largely unchanged for longer than most developers have been around.

**Review by evgeni — DISMISSED** [review]

File: locale/ca/foreman.po [hunk 1]
```diff
@@ -9440,6 +9500,9 @@ msgstr ""
 msgid "There was no active bridge interface found in libvirt, if it does not support listing, you can enter the bridge name manually (e.g. br0)"
 msgstr ""
 
+msgid "There was some issue deleting the report: ${error}"
```
Comment (ekohl) [inline locale/ca/foreman.po:9503]: Bleh, found another one. This comes from `webpack/assets/javascripts/react_app/components/HostDetails/Tabs/ReportsTab/helpers.js`

**Pull request merged**